core/src/components/slider/slider.ts

```diff
@@ -364,19 +365,17 @@ export function createSlider(config?: PropsConfig<SliderProps>): SliderWidget {
 	const {directive: sliderDirective, element$: sliderDom$} = createStoreDirective();
 	const {directive: minLabelDirective, element$: minLabelDom$} = createStoreDirective();
 	const {directive: maxLabelDirective, element$: maxLabelDom$} = createStoreDirective();
-
+	const resizeObserver = createResizeObserver();
 	const sliderResized$ = derived(
```

Comment:
I think you can remove this definition of `sliderResized$` and replace `sliderResized$` by `resizeObserver.dimensions$` everywhere.

Reply:
Thanks for the hint. I'll give it a try :)